### PR TITLE
NUTCH-3057 - Fix for index-arbitrary plugin improper retention and us…

### PR DESCRIPTION
**Pull request by CatChullain**
Fix for NUTCH-3057 where index-arbitrary plugin retained value for a field and erroneously set it to the next field declared in its config stanzas

### CODE REVIEW

**Comment by lewismc** [comment]
Thanks for reporting @CatChullain i didn’t catch this edge case either when reviewing or testing. 
Out curiosity what does your deployment look like? Local or deploy?

**Comment by lewismc** [comment]
Hi @CatChullain please send an email to me directly at lewismc [at] apache [dot] org. We would like to invite to to become a committer to the Nutch project.
Thank you

**Review by sebastian-nagel — APPROVED** [review]
+1 lgtm.

**Review by lewismc — APPROVED** [review]
+1 please merge @CatChullain

**Comment by CatChullain** [comment]
Thanks!

Merged now and resolved the Jira. Delete the fix's branch after the merge? Or wait till after the next public release?

**Comment by sebastian-nagel** [comment]
> Delete the fix's branch after the merge? Or wait till after the next public release?

Even if you delete the underlying branch, all changes and commit in the PR are still visible. So, it's safe to delete the branch now. Otherwise: since the branch is on your repository, it's solely your decision.

**Comment by CatChullain** [comment]
Thanks! Yeah, I see now that it says it's on my repo. I was thinking it was in the project's.

**Pull request merged**
